grpc-server-spring-boot-autoconfigure/src/main/java/net/devh/springboot/autoconfigure/grpc/server/GrpcServerFactory.java

```diff
@@ -4,21 +4,63 @@
 import net.devh.springboot.autoconfigure.grpc.server.codec.GrpcCodecDefinition;
 
 /**
- * User: Michael
- * Email: yidongnan@gmail.com
- * Date: 5/17/16
+ * A factory that can be used to create grpc servers.
+ *
+ * @author Michael (yidongnan@gmail.com)
+ * @sice 5/17/16
```

Comment:
```suggestion
 * @since 5/17/16
```
It looks like a spelling mistake here.